aws/cloudformation/components/www_redirect.yml.erb

```diff
@@ -1,12 +1,16 @@
+  <%
+    redirectedDomains = ["www.#{subdomain}"]
+    redirectedDomains += "www.#{CDO.pegasus_hostname}" if rack_env?(:production)
```

Comment:
We should probably avoid using `CDO.pegasus_hostname` because when this template is being used to provision a new environment that will have the value on the developer’s environment.

Reply:
wouldn't the rack_env check protect against that?

Reply:
Ah! True. Probably worth including a comment that, in general, `CDO.` properties are not safe to use in our build process because new environments are built from a development environment that may have different settings for hostname than the system they are provisioning.